components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/store/UserSessionStoreTest.java

```diff
@@ -239,6 +271,21 @@ public void testGetSessionId(String userId, String expectedSessionId) throws Exc
         }
     }
 
+    @Test(dataProvider = "getSessionAppsData", dependsOnMethods = {"testStoreUserSessionData"})
+    public void testStoreAppSessionData(String sessionId, String subject, int appID, String inboundAuth)
```

Comment:
Check how we can add an assertion.

Reply:
Here the related DAO method is not returning any results and we are mocking the Jdbc operations. If something goes wrong during the flow, the code will throw `DataAccessException` or `UserSessionException` and the test will fail. IMO we don't need an assertion here.
The flow is tested with assertions in integration tests.
https://github.com/wso2/product-is/pull/13289/files#diff-c4b18aa5c73ab10e8a6fcd5e410391a466713cba68342a691a75670e51d1c83eR135